draft-ietf-mops-streaming-opcons.md

```diff
@@ -448,6 +455,11 @@ All of this depends, of course, on the ability of a content provider to predict
 
 And as with other parts of the ecosystem, new technology brings new challenges.  For example, with the emergence of ultra-low-latency streaming, responses have to start streaming to the end user while still being transmitted to the cache, and while the cache does not yet know the size of the object.  Some of the popular caching systems were designed around cache footprint and had deeply ingrained assumptions about knowing the size of objects that are being stored, so the change in design requirements in long-established systems caused some errors in production.  Incidents occurred where a transmission error in the connection from the upstream source to the cache could result in the cache holding a truncated segment and transmitting it to the end user's device. In this case, players rendering the stream often had the video freeze until the player was reset.  In some cases the truncated object was even cached that way and served later to other players as well, causing continued stalls at the same spot in the video for all players playing the segment delivered from that cache node.
 
+
+## DNS
+
+TBD: More unique hostnames means more lookups and potential delays.
```

Comment:
This was part of Matt's original proposed text. I'll move it to a new issue like the section I suggested on segment sizing and keyframe intervals.

Reply:
Created https://github.com/ietf-wg-mops/draft-ietf-mops-streaming-opcons/issues/87